extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLogController.class.php

```diff
@@ -250,7 +257,7 @@ private function getSearchResults( $url, $query ) {
 		return $response;
 	}
 
-	private function appendRecordsFromResponse( array &$records, $response, array &$ipHash ) {
+	private function appendRecordsFromResponse( array &$records, $response, array &$domainCache ) {
```

Comment:
Why not have domainCache be member of the controller class so no need to pass it around?